infrahub_sdk/ctl/cli_commands.py

```diff
@@ -208,7 +208,6 @@ async def _run_transform(
         debug: Prints debug info to the command line
         repository_config: Repository config object. This is used to load the graphql query from the repository.
     """
-    branch = get_branch(branch)
```

Comment:
Please can you check if other commands are using get_branch(), I believe `validate_graphql` might be using it as well